runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -298,7 +298,7 @@ private boolean isFavoriteConfig(int world)
 	private void clearFavoriteConfig(int world)
 	{
 		configManager.unsetConfiguration(WorldHopperConfig.GROUP, "favorite_" + world);
-		panel.resetAllFavoriteMenus();
```

Comment:
This method is no longer used after this change, delete it from `WorldSwitcherPanel`.

Reply:
Got it, removed.